lib/Repository.js

```diff
@@ -491,7 +491,7 @@ class Repository extends Requestable {
     * @param {Requestable.callback} [cb] - will receive the pull request information
     * @return {Promise} - the promise for the http request
     */
-   updatePullRequst(number, options, cb) {
+   updatePullRequest(number, options, cb) {
```

Comment:
I'm guessing that you'll need to update some tests because of this. (Also this is a breaking change)